keystone_user.h

```diff
@@ -31,6 +31,8 @@
 #define UTM_FULL 4
 
 
+#if __riscv_xlen == 64
```

Comment:
Instead of this, you can try to define a type to deduplicate the code
e.g.,
```
#if __riscv_xlen == 64
  typedef __u64 param;
#elif __riscv_xlen == 32
  typedef __u32 param;
#endif
```

Reply:
Updated it